webapp/src/modules/address/selectors.js

```diff
@@ -33,30 +43,33 @@ export const getAddresses = createSelector(
       const contributions = (data[address].contributions || []).map(
         contribution => ({
           ...contribution,
-          district: districts[contribution.district_id]
+          district: allDistricts[contribution.district_id]
         })
       )
 
       // filter only open publications
       const publishedParcels = parcels.filter(parcel =>
-        isOpen(publications[parcel.publication_tx_hash])
+        isOpen(allPublications[parcel.publication_tx_hash])
       )
 
-      const mortgagedParcelsByAddress = mortgagedParcels.filter(
+      const mortgagedParcels = allMortgagedParcels.filter(
         parcel => parcel.mortgage.borrower === address
       )
 
+      const authorization = allAuthorizations[address]
+
       return {
         ...map,
         [address]: {
           ...data[address],
-          mortgagedParcels: mortgagedParcelsByAddress,
+          mortgagedParcels,
           parcels,
           parcelsById,
-          estates: estates.filter(e => e.owner === address),
+          estates,
           estatesById,
           contributions,
-          publishedParcels
+          publishedParcels,
+          authorization
```

Comment:
I added this here for completeness sake, but I'm not using it. @cazala do you think is better to keep it here anyways?

Reply:
👍